### PR TITLE
[ES-9800] Update deploy script to work with WigMix widgets

### PR DESCRIPTION
**Pull request by Niflnir**

### CODE REVIEW

File: init/index.js [hunk 1]
```diff
@@ -9,7 +9,7 @@
   const version = '$VERSION';
   const fieldsMappings = '$FIELDS_MAPPINGS';
   const widgetType = '$WIDGET_TYPE';
-  const initializer = context.visenzepswidget?.[widgetType]?.[`${version}`];
```
Comment (Niflnir) [inline init/index.js:12]: @wkurniawan07 this change will probably break your shopping assistant

File: init/index.js [hunk 2]
```diff
@@ -9,7 +9,7 @@
...
+  const initializer = context.visenzewigmixwidget?.[widgetType]?.[`${version}`];
```
Comment (Niflnir) [inline init/index.js:12]: @wkurniawan07 this change will probably break your shopping assistant widget

File: package.json [hunk 3]
```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "preinstall": "node -p \"'export const LIB_VERSION = \\'' + require('./package.json').version + '\\';'\" > src/version.ts",
     "build": "npm run preinstall && npm run build:camera-search && npm run build:similar-search && npm run build:search-results-page && npm run build:recommend-me && npm run build:more-like-this && npm run build:shop-the-look",
+    "build:staging": "export build=staging && npm run preinstall && npm run build:camera-search && npm run build:similar-search && npm run build:search-results-page && npm run build:recommend-me && npm run build:more-like-this && npm run build:shop-the-look",
```
Comment (wkurniawan07) [inline package.json:8]: Make it simpler:
```suggestion
    "build:staging": "export build=staging && npm run build",
```

Comment (Niflnir) [reply package.json:8]: Oh right didnt think about that :)

**Review by wkurniawan07 — APPROVED** [review]

**Pull request merged**